### PR TITLE
ACMS-3560: Support moderation dashboard major release version 3.x.

### PR DESCRIPTION
**Pull request by deepakmishra2**
**Motivation**

Fixes #ACMS-3560

Requirements
3.0.x
D10 ONLY, dropped support for charts v3 that hasn't had a release since 2022.

**Proposed changes**

update moderation dashboard to version 3.0

### CODE REVIEW

**Comment by chandan-singh7929** [comment]
Tests are failing @deepakmishra2

File: composer.json [hunk 1]
```diff
@@ -37,7 +37,8 @@
         "drupal/sitestudio_config_management": "dev-develop",
         "drupal/sitestudio_gin": "^1.0",
         "drush/drush": "^10 || ^11 || ^12",
-        "mnsami/composer-custom-directory-installer": "^2.0"
+        "mnsami/composer-custom-directory-installer": "^2.0",
+        "nnnick/chartjs": "^4.4"
```
Comment (apathak18) [inline composer.json:41]: Do we need this package in the require section of root composer.json?

**Review by deepakmishra2 — COMMENTED** [review]
Looks good to me

**Review by vishalkhode1 — APPROVED** [review]

File: composer.json (hunk 1 above)
Comment (deepakmishra2) [reply composer.json:41]: @chandan-singh7929 - i think we need to have 3.9 and 4.4 both, for the lower version to support

Comment (rajeshreeputra) [reply composer.json:41]: Agree with @apathak18 this is not required as acquia_cms_common has this dependency.
cc @vishalkhode1 @chandan-singh7929

**Pull request merged**